docs/Release/signing.md

```diff
@@ -0,0 +1,46 @@
+
+OSG Release Signing Information
+===============================
+
+Verifying OSG's RPMs
+--------------------
+
+We use a GPG key to sign our software packages. Normally `yum` and `rpm` transparently use the GPG signatures to verify the packages have not been corrupted and were created by us. You get our GPG public key when you install the `osg-release` RPM.
+
+If you wish to verify one of our RPMs manually, you can run:
+
+```console
+$ rpm --checksig -v <NAME.RPM>
+```
+
+For example:
+
+```console
+$ rpm --checksig -v globus-core-8.0-2.osg.x86_64.rpm 
+globus-core-8.0-2.osg.x86_64.rpm:
+    Header V3 DSA signature: OK, key ID 824b8603
+    Header SHA1 digest: OK (2b5af4348c548c27f10e2e47e1ec80500c4f85d7)
+    MD5 digest: OK (d11503a229a1a0e02262034efe0f7e46)
+    V3 DSA signature: OK, key ID 824b8603
+```
```

Comment:
It still looks broken to me, with the numbers I mentioned having different classes:

```html
<pre class="codehilite"><code class="language-console hljs vbnet">$ gpg --<span class="hljs-keyword">with</span>-fingerprint /etc/pki/rpm-gpg/RPM-GPG-<span class="hljs-keyword">KEY</span>-OSG
pub  <span class="hljs-number">1024</span>D/<span class="hljs-number">824</span>B8603 <span class="hljs-number">2011</span>-<span class="hljs-number">09</span>-<span class="hljs-number">15</span> OSG Software Team (RPM Signing <span class="hljs-keyword">Key</span> <span class="hljs-keyword">for</span> Koji Packages) &lt;vdt-support@opensciencegrid.org&gt;
      <span class="hljs-keyword">Key</span> fingerprint = <span class="hljs-number">6459</span> D9D2 AAA9 AB67 A251  FB44 <span class="hljs-number">2110</span> B1C8 <span class="hljs-number">824</span>B <span class="hljs-number">8603</span>
<span class="hljs-keyword">sub</span>  <span class="hljs-number">2048</span>g/<span class="hljs-number">28E5857</span>C <span class="hljs-number">2011</span>-<span class="hljs-number">09</span>-<span class="hljs-number">15</span></code></pre>
```